assets/blocks/course-outline/style.scss

```diff
@@ -107,6 +107,15 @@ $dark-gray: #1a1d20;
 		background-color: #DCDCDE;
 		border-radius: 50%;
 	}
+
+	&.completed::before {
+		content: '✓';
+		border: solid 1px #179942;
+		background-color: #1FD25A;
+		color: white;
+		line-height: 24px;
```

Comment:
I think it should be 22px to be exactly in the center.

Reply:
Fixed https://github.com/Automattic/sensei/pull/3643/commits/95859772b63c8439fd53061d644638c69b8216dc